youtube_dl/extractor/go90.py

```diff
@@ -70,6 +86,15 @@ def _real_extract(self, url):
                             'height': int_or_none(source.get('height')),
                             'tbr': int_or_none(source.get('bitrate')),
                         })
+
+                for caption in asset.get('caption_metadata', []):
+                    caption_url = caption.get('source_url')
+                    if not caption_url:
+                        continue
+                    subtitles.setdefault(caption.get('language', 'en'), []).append({
+                        'url': caption_url,
+                        'ext': determine_ext(caption_url, 'unknown_subtitle'),
```

Comment:
if the `ext` can't be detected than fallback to `vtt` as it's the `ext` that most likely to be.